andela/caret-bn-frontend
# #170336148 -  fix deployment error on heroku

#### What does this PR do?
Fix Deployment Issue on Heroku 
#### Description of Task to be completed?
- Remove CreateRequests.js from `/Requests` and place it in `/requests`
#### How should this be manually tested?
- Clone the repo
- Run npm install
- Ensure all files are present where they should be

#### Any background context you want to provide?
- Heroku failed to deploy develop
#### What are the relevant pivotal tracker stories?
#170336148
#### Screenshots (if appropriate)
N / A
#### Questions:
N / A
